api/rosetta/data_integration_test.go

```diff
@@ -35,14 +35,14 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/flow-go/model/flow"
+	invoker2 "github.com/optakt/flow-dps/invoker"
```

Comment:
Why do we need it to be called `invoker2` still? Isn't the other one removed now?

Reply:
Goland shenanigans 😬 Nice catch